sci-libs/rocFFT/rocFFT-4.3.0-r1.ebuild

```diff
@@ -22,9 +22,9 @@ DEPEND="${RDEPEND}
 
 BDEPEND="
 	test? ( dev-cpp/gtest dev-libs/boost
-	>=sci-libs/fftw-3
+	>=sci-libs/fftw-3 sys-libs/libomp )
 	>=dev-util/cmake-3.22
-)"
+"
```

Comment:
Explain in your comment message why this fixes https://bugs.gentoo.org/850937

Reply:
Done